spec/features/mailers/review_mailer_spec.rb

```diff
@@ -25,5 +25,11 @@
       expect(current_email).to have_content 'Alicja w krainie czarów'
       expect(current_email).to have_content 'Little about that story'
     end
+
+    scenario "akceptacja recenzji" do
```

Comment:
Trzeba to zrobić w obrębie scenariusza gdzie wysyła się prośbę o recenzję lub powtórzyć niezbędne kroki (takie jak kliknięcie na linku wysłania prośby). Inaczej odpowiedni mejl nie znajdzie się w skrzynce.